core-libs/setup/recipes/b2b/b2b-storefront.module.ts

```diff
@@ -1,24 +1,22 @@
 import { ModuleWithProviders, NgModule } from '@angular/core';
-import {
-  provideConfig,
-  provideDefaultConfig,
-  provideDefaultConfigFactory,
-} from '@spartacus/core';
+import { CostCenterModule, provideConfig, provideDefaultConfig, provideDefaultConfigFactory } from '@spartacus/core';
 import {
   CmsLibModule,
-  defaultB2bCheckoutConfig,
   defaultCmsContentConfig,
   layoutConfig,
   mediaConfig,
   StorefrontConfig,
-  StorefrontModule,
+  StorefrontModule
 } from '@spartacus/storefront';
-import { OrganizationModule } from '../organization.module';
+import { OrganizationModule } from '@spartacus/my-account/organization';
 import { defaultB2bOccConfig } from './config/b2b-occ-config';
 import { organizationLayoutConfig } from './config/organization-layout.config';
+import { defaultB2bCheckoutConfig } from './config/b2b-checkout-config';
+
 
 @NgModule({
   imports: [
+    CostCenterModule.forRoot(),
```

Comment:
I wonder if we can move this further down in the organization modules, as it's more in context then. i.e in `OrganizationModule`, `OrganizationComponentsModule` or `CostCenterComponentsModule`.

Reply:
This one is a CostCenter for b2b checkout